Sources/XMTP/Codecs/RemoteAttachmentCodec.swift

```diff
@@ -174,7 +174,7 @@ public struct RemoteAttachmentCodec: ContentCodec {
 			throw RemoteAttachmentError.invalidScheme("no scheme parameter")
 		}
 
-        if (!schemeString.starts(with: "https")) {
+        if (!schemeString.hasPrefix(RemoteAttachment.Scheme.https.rawValue)) {
```

Comment:
Changed this to hasPrefix since that seems to be more inline with how swift would do this.